ldap/servers/slapd/back-ldbm/dbimpl.c

```diff
@@ -397,6 +397,7 @@ const char *dblayer_op2str(dbi_op_t op)
     return str[idx];
 }
 
+<<<<<<< HEAD
```

Comment:
Probably a leftover from a merge.

Reply:
Indeed: bad conflict resolution when rebasing to main ...